### ISSUE TITLE
Question about OBSERVE state

### ISSUE THREAD

**Issue by mrgloom** [yenchenlin/DeepLearningFlappyBird#13]
What is purpose of number of OBSERVE steps > size of REPLAY_MEMORY?

**Comment by yenchenlin1994** [comment]
Hello @mrgloom ,
I set OBSERVE steps so big just for demo purpose :smile: 

If you are trying to reproduce the model,
I've added [a section](https://github.com/yenchenlin1994/DeepLearningFlappyBird/blob/master/README.md#how-to-reproduce) about that.

**Comment by yenchenlin1994** [comment]
Hi @mrgloom ,
If above comments have answered your question, would you please close this issue?
Thanks!

**Comment by mrgloom** [comment]
I'm still not sure how number of OBSERVE timesteps estimated, it's just arbitary number BATCH < OBSERVE < REPLAY_MEMORY ?

Also what if I can't do all 3000000 at one time, how training can be continued? Just set OBSERVE to same value, load CNN weights, and set EXPLORE = (3000000 - steps_already_trained) ?

**Comment by yenchenlin1994** [comment]
Hello @mrgloom 

1. arbitary number BATCH < OBSERVE <= REPLAY_MEMORY

However, I set it according to the reference paper and empirical result.

2. Yes

**Comment by mrgloom** [comment]
Also is there something special about OBSERVE state, for example should bird pass through a pipe at least once during this state or it's not necessary?
Or OBSERVE state just used to init replay memory?

Also I run 2 training cases (about 150000 timesteps) one with recommended parameters and another with no EXPLORE state at all (I set FINAL_EPSILON and INITIAL_EPSILON to 0)

I found that without EXPLORE state it also learn to play fine, but my intuition about this that it will choose more long routes and this will lead to more risky playing, and with random actions at each timestep with small probability model learn to play more safely(so it's some kind of regularization?). 

What my intuition can't understand is that how model learns to play game if during OBSERVE state bird do not pass any pipes.
